paasta_tools/contrib/get_running_task_allocation.py

```diff
@@ -248,36 +172,15 @@ def get_kubernetes_task_allocation_info(
                     host_ip=info.get("host_ip"),
                     git_sha=info.get("git_sha"),
                     config_sha=info.get("config_sha"),
-                    mesos_container_id=None,
```

Comment:
note: might be worth checking that the consumers of this will handle this field not existing at all anymore (pretty sure only splunk reads this and that it will gracefully handle things tho)

Reply:
looks like the savedsearch might do exactly the opposite of what we want if the mesos_container_id is missing. It's currently checking specifically for "null".